### ISSUE TITLE
Is it possible to use gem with custom api server?

### ISSUE THREAD

**Issue by yuri-val** [telegram-bot-rb/telegram-bot#146]
Using a Local Bot API Server
https://core.telegram.org/bots/api#using-a-local-bot-api-server

**Comment by printercu** [comment]
Wow, this is something new. At this time there is no configuration option :( I'll add it soon.

Until this it's possible to override the const:
```ruby
Telegram::Bot::Client.class_eval do 
  remove_const(:URL_TEMPLATE)
  # original is 'https://api.telegram.org/bot%<token>s/'
  self::URL_TEMPLATE = 'https://custom.host/bot%<token>s/'
end
```

Make sure to apply this patch before any bot is instantiated.

**Comment by yuri-val** [comment]
Thx. Using your gem is very convenient

**Comment by printercu** [comment]
Closed by #147
